fuzz/src/lib.rs

```diff
@@ -186,6 +664,42 @@ pub fn xml_to_events(xml: &str) -> anyhow::Result<Vec<Event>> {
     Ok(events)
 }
 
+pub fn normalize_pandoc(events: Vec<Event<'_>>) -> Vec<Event<'_>> {
```

Comment:
I think this is cool! I would rename `normalize` below to `normalize_commonmarkjs` or similar.